File: Javascript/fizzBuzz.js
function range(inicio, fin) {
    const NUMEROS = [];
    for (let $i = inicio ; $i < (fin + 1); $i++) {
        NUMEROS.push($i)
    }
    return NUMEROS;
}

const NUMEROS = range(1,100);

for (const num of NUMEROS) {
    var $divTres = num % 3 == 0;
    var $divCinco = num % 5 == 0;

    if ($divTres && $divCinco) {
        console.log("fizzbuzz");
    } else if ($divTres) {
        console.log("fizz");
    } else if ($divCinco) {
        console.log("buzz");
    } else {
        console.log(num);
    }
}

File: Java/fizzbuzz.java
package Java;

/**
 * fizzBuzz
 */
public class fizzbuzz {
    public static void main(String[] args) {
        for (int i = 0; i <= 100; i++) {
            boolean divTres = i % 3 == 0;
            boolean divCinco = i % 5 == 0;
            if (divTres && divCinco) {
                System.out.println("fizzbuz");
            } else if (divTres) {
                System.out.println("fizz");
            } else if (divCinco) {
                System.out.println("buzz");
            } else {
                System.out.println(i);
            }
        }
    }
}
